ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/RootPeers.hs

```diff
@@ -109,17 +112,33 @@ jobReqPublicRootPeers PeerSelectionActions{ requestPublicRootPeers
 
     job :: m (Completion m peeraddr peerconn)
     job = do
-      (results, ttl) <- requestPublicRootPeers numExtraAllowed
+      (results, ttl) <- requestPublicRootPeers AllLedgerPeers numExtraAllowed
       return $ Completion $ \st now ->
-        let newPeers         = results `Map.withoutKeys` LocalRootPeers.keysSet (localRootPeers st)
-                                       `Map.withoutKeys` publicRootPeers st
-                                       `Map.withoutKeys` bigLedgerPeers st
-            publicRootPeers' = publicRootPeers st <> Map.keysSet newPeers
-            knownPeers'      = KnownPeers.insert
-                                 -- When we don't know about the PeerSharing information
-                                 -- we default to NoPeerSharing
-                                 (Map.map (\(a, b) -> (Just PeerSharingDisabled, Just a, Just b)) newPeers)
```

Comment:
Yes, that
s one reason for the change, instead of explicitly passing `Just PeerSharingDisabled` which should be the default we only pass the value we want to overwrite and `Nothing` defaults to the default value